packages/cli/src/credentials/credentials.service.ts

```diff
@@ -113,13 +113,6 @@ export class CredentialsService {
 				);
 			}
 
-			credentials.forEach((c) => {
```

Comment:
Did you ensure that this function is not called in any other place where this might cause issues?

Reply:
I did. this is called only from two places.
1. Credentials controller: The controller takes care of doing this now.
2. Workflows controller: uses the returned value for permission checking, and does not serialize the data back to the frontend.